### ISSUE TITLE
filter list by rank: research issue

### ISSUE THREAD

**Issue by arlin** [phylotastic/phylotastic-portal#268]
User BS writes "In the search results where one selects which matches to use for the tree, it would be nice to separate out Latin binomials from the higher taxa. E.g., it would be nice to be able to select all families (-idae), all orders (-iformes) or all Genus/species pairings."

We can get the rank from OTT results (rank = Phylum, Class, Order, Family, Genus, Species). 

I think the typical use-case will be that the user will want only species, or they will want to eliminate higher taxa above a certain rank.  

Ideally, we would have a "Remove higher taxa" with a pull-down for Phylum, Class, Order, Family, Genus, Species.  However, this will be difficult because many taxa do not have ranks, or their have intermediate ranks like super-family, tribe, etc.  

So, this one requires some research to find out what we can do easily.  Can we identify which names are species or lower, and separate them from genera and higher?  Maybe we have to ask OT people to advise.

**Comment by hdliv** [comment]
@arlin I agree with BS.  This was actually similar to an issue I brought up in one of our meetings when I was talking about everything that is scraped from a web or paper.  We should talk to OT about this.  @bomeara maybe you have a suggestion
